apps/test/unit/StudioAppTest.js

```diff
@@ -185,6 +186,75 @@ describe('StudioApp', () => {
     });
   });
 
+  describe('loadLibraryBlocks', () => {
+    const initialConfig = {
+      level: {
+        codeFunctions: {preExistingFunction: null}
+      },
+      dropletConfig: {
+        additionalPredefValues: ['preExistingValue'],
+        blocks: ['preExistingBlock']
+      }
+    };
+
+    it('given no libraries, leaves the config unchanged', () => {
```

Comment:
Is this test the same as the next one?

Reply:
Oops. Looks like I forgot to update the second one. Thanks for catching that.